cherro-front/package.json

```diff
@@ -38,6 +38,14 @@
   },
   "jest": {
     "clearMocks": true
+  },
+  "devDependencies": {
+    "husky": "^4.3.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test",
```

Comment:
We should avoid this as a pre-commit hook. It will complicate development.

Reply:
pre-commint deleted